cnxarchive/tests/test_database.py

```diff
@@ -1539,6 +1539,7 @@ def test_new_module_wo_uuid(self, cursor):
         self.assertEqual(uuid_, controls_uuid)
         self.assertEqual(license_id, controls_license_id)
 
+
```

Comment:
Done!
I was trying to use the Atom editor, seems it like to take a little more liberty with changing things than I would like - sorry about that!

> On Aug 31, 2015, at 4:28 PM, Karen Chan <notifications@github.com> wrote:
> 
> In cnxarchive/tests/test_database.py <https://github.com/Connexions/cnx-archive/pull/344#discussion_r38361961>:
> 
> > @@ -1539,6 +1539,7 @@ def test_new_module_wo_uuid(self, cursor):
> >          self.assertEqual(uuid_, controls_uuid)
> >          self.assertEqual(license_id, controls_license_id)
> >
> > +
> Could you remove the extra blank lines around methods?
> There should only be one blank line between methods.
> 
> Running pep8 gives:
> 
> cnxarchive/tests/test_database.py:1543:5: E303 too many blank lines (2)
> cnxarchive/tests/test_database.py:1602:5: E303 too many blank lines (2)
> cnxarchive/tests/test_database.py:1642:5: E303 too many blank lines (2)
> I know our code is not yet pep8 compliant, but we're trying to be. :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Connexions/cnx-archive/pull/344/files#r38361961>.
>